contrib/ohi-release-notes/rt-dictionary.yml

```diff
@@ -0,0 +1,31 @@
+# This file is a dictionary used by the [link-dependencies](https://github.com/newrelic/release-toolkit/tree/main/link-dependencies) action.
+dictionary:
+  newrelic/infrastructure-bundle: "https://github.com/newrelic/infrastructure-bundle/releases/tag/{{.To}}"
+  newrelic/infrastructure: "https://github.com/newrelic/infrastructure-agent/releases/tag/{{.To}}"
+  newrelic/nri-apache: "https://github.com/newrelic/nri-apache/releases/tag/{{.To}}"
+  newrelic/nri-cassandra: "https://github.com/newrelic/nri-cassandra/releases/tag/{{.To}}"
+  newrelic/nri-consul: "https://github.com/newrelic/nri-consul/releases/tag/{{.To}}"
+  newrelic/nri-couchbase: "https://github.com/newrelic/nri-couchbase/releases/tag/{{.To}}"
+  newrelic/nri-discovery-kubernetes: "https://github.com/newrelic/nri-discovery-kubernetes/releases/tag/{{.To}}"
+  newrelic/nri-docker: "https://github.com/newrelic/nri-docker/releases/tag/{{.To}}"
+  newrelic/nri-ecs: "https://github.com/newrelic/nri-ecs/releases/tag/{{.To}}"
+  newrelic/nri-elasticsearch: "https://github.com/newrelic/nri-elasticsearch/releases/tag/{{.To}}"
+  newrelic/nri-f5: "https://github.com/newrelic/nri-f5/releases/tag/{{.To}}"
+  newrelic/nri-haproxy: "https://github.com/newrelic/nri-haproxy/releases/tag/{{.To}}"
+  newrelic/nri-jmx: "https://github.com/newrelic/nri-jmx/releases/tag/{{.To}}"
+  newrelic/nri-kafka: "https://github.com/newrelic/nri-kafka/releases/tag/{{.To}}"
+  newrelic/nri-memcached: "https://github.com/newrelic/nri-memcached/releases/tag/{{.To}}"
+  newrelic/nri-mongodb: "https://github.com/newrelic/nri-mongodb/releases/tag/{{.To}}"
+  newrelic/nri-mssql: "https://github.com/newrelic/nri-mssql/releases/tag/{{.To}}"
+  newrelic/nri-mysql: "https://github.com/newrelic/nri-mysql/releases/tag/{{.To}}"
+  newrelic/nri-nagios: "https://github.com/newrelic/nri-nagios/releases/tag/{{.To}}"
+  newrelic/nri-nginx: "https://github.com/newrelic/nri-nginx/releases/tag/{{.To}}"
+  newrelic/nri-oracledb: "https://github.com/newrelic/nri-oracledb/releases/tag/{{.To}}"
+  newrelic/nri-postgresql: "https://github.com/newrelic/nri-postgresql/releases/tag/{{.To}}"
+  newrelic/nri-prometheus: "https://github.com/newrelic/nri-prometheus/releases/tag/{{.To}}"
+  newrelic/nri-rabbitmq: "https://github.com/newrelic/nri-rabbitmq/releases/tag/{{.To}}"
+  newrelic/nri-redis: "https://github.com/newrelic/nri-redis/releases/tag/{{.To}}"
+  newrelic/nri-varnish: "https://github.com/newrelic/nri-varnish/releases/tag/{{.To}}"
+  newrelic/nri-winservices: "https://github.com/newrelic/nri-winservices/releases/tag/{{.To}}"
+  newrelic/nrjmx: "https://github.com/newrelic/nrjmx/releases/tag/{{.To}}"
+  newrelic/prometheus-exporter-supervisor: "https://github.com/newrelic/prometheus-exporter-supervisor/releases/tag/{{.To}}"
```

Comment:
NIT: I have just realised that we do not have Prometheus exporters packages: https://github.com/newrelic/newrelic-prometheus-exporters-packages

Does it even make sense to have it?

Reply:
i don't think so, it shouldn't be a dependency anywhere. We could add it in the future if needed